packages/web-app/src/components/appli/Dashboard/components/CavesData/InlineData.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Typography, Box } from '@material-ui/core';
+
+const StyledIcon = withStyles(() => ({
+  root: {
+    width: '25px',
+    height: '25px'
+  }
+}))(Box);
```

Comment:
This should work and is IMO a bit easier to read

```suggestion
const StyledIcon =styled.div`
  width: '25px',
  height: '25px'
`
```

or 
```
const Styled(Box)`
  width: '25px',
  height: '25px'
`
```

Same for above and other places where we use `withStyles`

Reply:
We used `withStyles()` because we found that it was commonly used in the code. We tried the 2 suggestions you made and the styles are not applied to our images so we would like to keep our current implementation if it's possible.